pkgs/applications/science/misc/openmvs/default.nix

```diff
@@ -18,12 +38,34 @@ stdenv.mkDerivation rec {
   };
 
   # SSE is enabled by default
-  cmakeFlags = lib.optional (!stdenv.isx86_64) "-DOpenMVS_USE_SSE=OFF";
+  cmakeFlags = [
+    "-DOpenMVS_ENABLE_TESTS=OFF"
```

Comment:
would it be possible to build the tests, to run them with
```
doCheck = true;
checkPhase = ''
something
'';
```
just not to install them?
(but it that does not work for example because tests fail in the sandbox, current code is ok)

Reply:
Great suggestions, I added a check and installCheck phase!